packages/material-ui/src/styles/experimentalStyled.test.js

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import { expect } from 'chai';
+import { createClientRender, screen } from 'test/utils';
+import createMuiTheme from './createMuiTheme';
+import styled from './experimentalStyled';
+import ThemeProvider from './ThemeProvider';
+
+describe('experimentalStyled', () => {
+  const render = createClientRender();
+  it('should work', () => {
+    const Div = styled('div')({
+      width: '200px',
+    });
+
+    render(<Div data-testid="component">Test</Div>);
+
+    const style = window.getComputedStyle(screen.getByTestId('component'));
+    expect(style.getPropertyValue('width')).to.equal('200px');
+  });
+
+  it('should use defaultTheme if no theme is provided', () => {
+    const Div = styled('div')((props) => ({
+      backgroundColor: props.theme.palette.background.paper,
+    }));
+
+    render(<Div data-testid="component">Test</Div>);
+
+    const style = window.getComputedStyle(screen.getByTestId('component'));
+    expect(style.getPropertyValue('background-color')).to.equal('rgb(255, 255, 255)'); // white
```

Comment:
nit: Can we use something other than colors or font size? These CSS properties are brittle inside JSDOM. Since it's a test for basic functionality it should be runnable inside JSDOM (assuming this is the preferred test environment).

Reply:
Sure, let me refactor by using margins :)